### DISCUSSION TITLE
Editable notes

### DISCUSSION THREAD

**Discussion by webenadam** [alangrainger/share-note#105] [Vote for upcoming features]
Being able to share a note and let others add tasks, remove tasks and so on, would make this tool amazing for collaborating with others in a easy way.

**Comment by NA0341** [comment]
The idea is great ~ but the technical implementation might be rather difficult.

Because for Things to Not become a Mess ~ there must be a Function which Syncs `Changes on the Site` Back to the Obsidian Vault that published the Note.

In other Words: This would need to be a Web Frontend to Edit the Note ~ as well as a Sync Backend to transfer the changes back to the Original.

It is doable though.
The Plugin could just ask the Server for Changes when you open the Note.

**Comment by alangrainger** [comment]
Thank you for the suggestion, but this isn't in scope for the plugin.

I suggest you use one of the many collaboration tools which already exist. Here is a useful list of open source and privacy oriented options:

https://www.privacytools.io/productivity
